src/swap/SwapScreen.tsx

```diff
@@ -684,14 +698,14 @@ export function SwapScreen({ route }: Props) {
     {
       fieldType: Field.FROM,
       tokens: swappableFromTokens,
-      title: t('swapScreen.swapFromTokenSelection'),
+      title: t('swapScreen.selectTokenTitle'),
       filterChips: filterChipsFrom,
       origin: TokenPickerOrigin.SwapFrom,
     },
     {
       fieldType: Field.TO,
       tokens: swappableToTokens,
-      title: t('swapScreen.swapToTokenSelection'),
+      title: t('swapScreen.selectTokenTitle'),
```

Comment:
since these are the same now, it can probably be passed directly to the component and removed from this map